custom-metrics-stackdriver-adapter/pkg/adapter/translator/utils/filter_builder.go

```diff
@@ -19,12 +20,18 @@ type Schema struct {
 }
 
 const (
-	PodType        = "k8s_pod"
-	PrometheusType = "prometheus_target"
-	LegacyType     = "<not_allowed>"
+	PodSchemaKey        = "pod"               // PodSchemaKey is the key to use pod type filter schema.
```

Comment:
Not sure if GoDoc picks these comments up correctly. If they are above the consts it will, but just a minor nit.

Reply:
yep, GoDoc renders it, at least I use vscode, and it works for me. I think for variable type, put the doc after it instead of above it makes it a little bit neater, how do you think 🤔

Reply:
Perfect, thx